### PR TITLE
Autohide scroll memory.

### PR DESCRIPTION
**Pull request by pelatx**
Hi,

This tries to fix issue #106.

It is working for me. Let me know if you find any errors @Hirse.

### CODE REVIEW

File: src/Autohide.js [hunk 1]
```diff
@@ -94,6 +95,7 @@ define(function (require, exports, module) {
             removeSidebarPlusTransition();
             hidePlaceholder();
             OutlineManager.showOutline();
+            $("#outline-list").scrollTop(scrollBackup);
```
Comment (Hirse) [inline src/Autohide.js]: Does it make sense to check if `scrollBackup` is set to a value before this call or is jQuery taking care of that?

File: src/Autohide.js [hunk 2]
```diff
@@ -14,6 +14,7 @@ define(function (require, exports, module) {
     var $mainView = $(".main-view");
     var isExposed = false;
     var sidebarPlusTransitionRemoved = false;
+    var scrollBackup;
```
Comment (Hirse) [inline src/Autohide.js]: Could you maybe rename this to `scrollTop`, `previousScrollTop`, or something similar?

File: src/Autohide.js (hunk 1 above)
Comment (pelatx) [reply src/Autohide.js]: I think it is never read without having been previously saved, due to Autohide's own operation.

But I think checking it does not hurt.

File: src/Autohide.js (hunk 2 above)
Comment (pelatx) [reply src/Autohide.js]: First I wrote `scrollTop` and then dismissed it as redundant, hehe

Renamed.

**Pull request merged**